.yarnrc

```diff
@@ -1,2 +1,7 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
 ignore-engines true
-yarn-path ".yarn/releases/yarn-1.19.1.js"
+lastUpdateCheck 1674843936353
+yarn-path ".yarn/releases/yarn-1.21.1.js"
```

Comment:
are these changes, and the .yarn/releases changes meant to be in the form PR? It's massive.

Reply:
Reverted.